simvue/executor.py

```diff
@@ -57,6 +57,7 @@ def add_process(
         executable: typing.Optional[str]= None,
         script: typing.Optional[str]= None,
         input_file: typing.Optional[str]= None,
+        completion_callback: typing.Optional[typing.Callable]=None,
```

Comment:
Specify format of expected function in Typing, `(status_code: int, std_out: str, std_err: str)`